circle.yml

```diff
@@ -1304,11 +1313,29 @@ jobs:
     steps:
       - restore_cached_workspace
       - restore_cached_system_tests_deps
-      - run:
-          name: Run v8 integration tests
-          command: |
-            source ./scripts/ensure-node.sh
-            yarn test-integration --scope @tooling/packherd
+      - unless:
+          condition:
+            or:
+              - equal: [ *linux-arm64-executor, << parameters.executor >> ] # TODO: Figure out how to support linux-arm64 when we get to linux arm64 build: https://github.com/cypress-io/cypress/issues/23557 
+              - equal: [ *windows-executor, << parameters.executor >> ] # TODO: Figure out how to support windows when we support the windows build: https://github.com/cypress-io/cypress/issues/23122
+          steps:
+            - run:
+                name: Run v8 integration tests
+                command: |
+                  source ./scripts/ensure-node.sh
+                  yarn test-integration --scope "'@tooling/{packherd,v8-snapshot,electron-mksnapshot}'"
+      - when:
+          condition:
+            or:
+              - equal: [ *linux-arm64-executor, << parameters.executor >> ]
+              - equal: [ *windows-executor, << parameters.executor >> ]
+          steps:
+            - run:
+                name: Run v8 integration tests
+                command: |
+                  source ./scripts/ensure-node.sh
+                  yarn test-integration --scope "'@tooling/packherd'"
+      # Windows and arm executors don't support verify-mocha-results
```

Comment:
This `unless` block can be removed now, I think, since we check for `CI_DOCKER` here now, which is only set in Docker Linux: https://github.com/cypress-io/cypress/blob/b615e8ac30b681b4dc8485afe9814ff453da1fbf/scripts/verify-mocha-results.js#L86-L87

Reply:
Updated here: [`9ac5768` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/9ac57680068b11b55ee9c565d3df2f0c80892f0a)